sdcm/ec2_client.py

```diff
@@ -353,3 +356,21 @@ def get_instance_by_private_ip(self, private_ip):
             raise GetInstanceByPrivateIpError("Cannot find instance by private ip: %s" % private_ip)
 
         return self.get_instance(instances['Reservations'][0]['Instances'][0]['InstanceId'])
+
+    def add_placement_group_name_param(self, boto3_params):
+        sct_params = SCTConfiguration()
+        if sct_params.get("use_placement_group"):
+            config = TestConfig()
+            placement_group_name = f"{sct_params.get('user_prefix')}-placement_group-{config.test_id()[:8]}"
```

Comment:
I'd rather reused `placement_group_name`' cluster's property, e.g. by providing it as function argument (as function doesn't have access to cluster)

Reply:
another solution could be adding this param into `TestConfig` - so it could be reused by cluster and this function

Reply:
updated to reuse logic from PlacementGroup class